src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Attr.java

```diff
@@ -3572,7 +3573,7 @@ public Env<AttrContext> lambdaEnv(JCLambda that, Env<AttrContext> env) {
                     if (clinit == null) {
                         Type clinitType = new MethodType(List.nil(),
                                 syms.voidType, List.nil(), syms.methodClass);
-                        clinit = new MethodSymbol(STATIC | SYNTHETIC | PRIVATE,
+                        clinit = new MethodSymbol(BLOCK | STATIC | SYNTHETIC | PRIVATE,
```

Comment:
Added missing `BLOCK`, so that we're now consistent with what the rest of the code (not only `LambdaToMethod`) expects.